src/Microsoft.SqlTools.ServiceLayer/TableDesigner/Contracts/Requests/GeneratePreviewReportRequest.cs

```diff
@@ -7,6 +7,18 @@
 
 namespace Microsoft.SqlTools.ServiceLayer.TableDesigner.Contracts
 {
+    /// <summary>
+    /// The service request to generate preview report describing the changes.
+    /// </summary>
+    public class GeneratePreviewReportResult 
+    {
+        // The script generated in plaintext or markdown
+        public string Script;
+
+        // format of the string (plaintext or mardown)
+        public string Format;
```

Comment:
I'd say this should be optional - and default to plaintext if none is specified.

I'd also suggest that this be `MimeType` instead - that's the standard way of controlling how content should be rendered. For markdown it'd be `text/markdown`